web/router.ex

```diff
@@ -33,19 +33,24 @@ defmodule Entice.Web.Router do
   scope "/api", Entice.Web do
     pipe_through :api
 
-    post "/login",            AuthController, :login
-    post "/logout",           AuthController, :logout
+    post    "/login",             AuthController, :login
+    post    "/logout",            AuthController, :logout
 
-    get  "/char",             CharController, :list
-    post "/char",             CharController, :create
+    get     "/char",              CharController, :list
+    get     "/char/account/:name" CharController, :get_account
```

Comment:
Missing `,` after the route, and maybe this would be better in AccountController? And with a route like "/account/by_char_name" and "char_name" as a parameter, similar to how it is done in the TokenController :)